web/src/engine/websites/ReaperScans.ts

```diff
@@ -48,66 +41,44 @@ export default class extends DecoratableMangaScraper {
     }
 
     public override ValidateMangaURL(url: string): boolean {
-        return new RegExpSafe(`^${this.URI.origin}/comics/[^/]+$`).test(url);
+        return new RegExpSafe(`^${this.URI.origin}/series/[^/]+$`).test(url);
     }
 
     public override async FetchManga(provider: MangaPlugin, url: string): Promise<Manga> {
-        return this.interactionTaskPool.Add(async () => Common.FetchMangaCSS.call(this, provider, url, 'h1.text-xl'), Priority.Normal);
+        const slug = url.match(/\/series\/([^/]+)/)[1];
```

Comment:
No need for regex. Based on `ValidateMangaURL` the last segment must be the slug, therefore this can be simplified to `url.split('/').at(-1)` or  `url.split('/').pop()`.